src/GitHub.Logging/Logging/LogManager.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using GitHub.Info;
+using Serilog;
+using Serilog.Core;
+
+namespace GitHub.Logging
+{
+    public static class LogManager
+    {
+        static Logger CreateLogger()
+        {
+            var logPath = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+                ApplicationInfo.ApplicationName,
+                "extension.log");
+
+            const string outputTemplate =
+                "{Timestamp:yyyy-MM-dd HH:mm:ss.fff}|{Level}|Thread:{ThreadId}|{SourceContext}|{Message}{NewLine}{Exception}";
```

Comment:
Many default output templates now use `{Message:lj}` to turn on un-quoted string literals, and to use JSON rather than Serilog's display format for any serialized structured data. Small tweak, but can look a bit nicer :-)